app/src/main/java/com/ikarabulut/GameDisplay.java

```diff
@@ -64,4 +70,9 @@ public String deadSymbolPrompt() {
         return deadSymbolPrompt;
     }
 
+    public String rulesPrompt() {
+        System.out.println(rulesPrompt);
+        return rulesPrompt;
```

Comment:
I'm not positive, but I don't think any of your prompt methods need to return the string. I think they just need to print the prompt/question to system out.

Reply:
Yeah this is correct. I need to update my tests to use System.setOut so I don't rely on a return and just test what is printed to the console

Reply:
Just updated the tests